Makefile

```diff
@@ -15,9 +15,12 @@ help:
 	@echo "    push-check:         Replicates the github workflow checks as close as possible"
 	@echo "                        (do this before pushing!)"
 
+VERSION="NotUsed"
```

Comment:
(nitpick) I personally would not include this yet if it is always set to "NotUsed" and only introduce it when it is used (YAGNI and all that :)